src/qt/optionsmodel.cpp

```diff
@@ -54,14 +54,15 @@ void OptionsModel::Init(bool resetSettings)
     // These are Qt-only settings:
 
     // Window
-    if (!settings.contains("fHideTrayIcon"))
-        settings.setValue("fHideTrayIcon", false);
-    fHideTrayIcon = settings.value("fHideTrayIcon").toBool();
-    Q_EMIT hideTrayIconChanged(fHideTrayIcon);
+    if (!settings.contains("bShowTrayIcon")) {
```

Comment:
nit, is there a reason to change setting key? No big deal in this case, but ideally "old" key would be migrated - if you wish to rename the key.

Reply:
It is not just renaming here. `fHideTrayIcon` and `bShowTrayIcon` are different semantically. As the `Bitcoin-Qt.conf` is written in the human readable format, I'd prefer to introduce a new setting key to avoid any possible confusing.

An old key migration sounds good to me. But is it worth to add more code for such a non-critical key?

Reply:
> It is not just renaming here. `fHideTrayIcon` and `bShowTrayIcon` are different semantically.

Of course, you would have to take the !fHideTrayIcon.

> An old key migration sounds good to me. But is it worth to add more code for such a non-critical key?

I've started with "nit" 😅

Reply:
>It is not just renaming here. fHideTrayIcon and bShowTrayIcon are different semantically.

How?

>As the Bitcoin-Qt.conf is written in the human readable format, I'd prefer to introduce a new setting key to avoid any possible confusing.

That's not a good reason IMO

Reply:
> > It is not just renaming here. fHideTrayIcon and bShowTrayIcon are different semantically.
> 
> How?

The meaning of `fHideTrayIcon` is opposite to `bShowTrayIcon`, no?

> 
> > As the Bitcoin-Qt.conf is written in the human readable format, I'd prefer to introduce a new setting key to avoid any possible confusing.
> 
> That's not a good reason IMO

What do you suggest?

Reply:
> The meaning of fHideTrayIcon is opposite to bShowTrayIcon, no?

It's a strict opposite.

>What do you suggest?

Leave `fHideTrayIcon` alone and just change the UI presentation.

Reply:
[Updated](https://github.com/bitcoin-core/gui/pull/115#issuecomment-727605555).